wasmtime/_sharedmemory.py

```diff
@@ -0,0 +1,149 @@
+from . import _ffi as ffi
+from ctypes import *
+import ctypes
+import typing
+from wasmtime import MemoryType, WasmtimeError, Engine
+from ._store import Storelike
+
+
+
+class SharedMemory:
+    _sharedmemory: ffi.wasmtime_sharedmemory_t
```

Comment:
Could this have a superclass of `Managed` like most other types in the bindings?

Reply:
Done!
Let me know if I did something wrong